imports/both/i18n/en/faq.json

```diff
@@ -1,13 +1,21 @@
 {
   "faq" : [
     {
-      "heading" : "faq heading",
-      "text" : "faq text"
+      "heading" : "First question",
+      "text" : "This is always the 1st question asked. Actually you'd be amazed how many people living on the streets have a smart phone as people often donate their old phones to charities and shelters, and they all know where to find wifi." 
     },
     {
-      "heading" : "faq heading",
-      "text" : "faq text"
+      "heading" : "Cozy sports for Wi-fi!",
```

Comment:
Should be `Cozy spots`

Reply:
Thanks for the review. I’ll fix this in a bit.